packages/ag-charts-community/src/chart/update/dataWindowProcessor.ts

```diff
@@ -1,27 +1,37 @@
 import { ChartUpdateType } from '../chartUpdateType';
 import type { DataService } from '../data/dataService';
 import type { ZoomManager, ZoomState } from '../interaction/zoomManager';
+import type { LayoutCompleteEvent, LayoutService } from '../layout/layoutService';
 import type { UpdateService } from '../updateService';
 import type { AxisLike, ChartLike, UpdateProcessor } from './processor';
 
+type Window = {
+    min: Date;
+    max: Date;
+};
+
 export class DataWindowProcessor<D extends object> implements UpdateProcessor {
     private dirtyZoom = false;
     private dirtyDataSource = false;
-    private lastAxisZooms = new Map<string, ZoomState>();
+    private dirtyLayout = false;
+    private lastChartSize: { width: number; height: number } | undefined = undefined;
```

Comment:
I feel like we maybe mixing too many concerns here, the `DataWindowProcessor` shouldn't be concerned about layout state, or embedding logic about how to determine if another layout is likely to happen?